src/lib/components/HttpErrors/HttpErrors.js

```diff
@@ -0,0 +1,64 @@
+import Overridable from 'react-overridable';
+import React from 'react';
+import HttpError from './HttpError';
+
+function UnauthorizedComponent(props) {
+  return (
+    <Overridable id="Unauthorized.layout" {...props}>
+      <HttpError
+        title="Unauthorized"
+        message="You are not authorized to access this resource."
+        icon="ban"
+      />
+    </Overridable>
+  );
+}
+
+function ForbiddenComponent(props) {
+  return (
+    <Overridable id="Forbidden.layout" {...props}>
+      <HttpError
+        title="Access Forbidden"
+        message="You do not have the sufficient privileges to access this resource."
+        icon="ban"
+      />
+    </Overridable>
+  );
+}
+
+function NotFoundComponent(props) {
+  return (
+    <Overridable id="NotFound.layout" {...props}>
+      <HttpError
+        title="Page Not Found"
+        message="The requested page could not be found."
+        icon="compass outline"
+      />
+    </Overridable>
+  );
+}
+
+function InternalServerErrorComponent(props) {
+  return (
+    <Overridable id="InternalServerError.layout" {...props}>
+      <HttpError
+        message="Internal Server Error"
```

Comment:
There is a mix here, between the title message and that is an interesting icon you got there :D